docs/after/old-what-to-bring.md

```diff
@@ -77,7 +77,7 @@ title: What to Bring?
       <th scope="row" rowspan="8">Other Documents</th>
       <td>1</td>
       <td>Tuition Grant Agreement</td>
-      <td>If ditandatanganin di Indo, with notaris. More info <a href="tg">here</a>.</td>
+      <td>If ditandatanganin di Indo, with notaris. More info <a href="/docs/after/tg">here</a>.</td>
```

Comment:
Kenapa bahasanya english + indo ya HAHAHA

Reply:
Hahaha, oh actually this file is not used anymore, but kept just in case it is needed in the future.